Session.js

```diff
@@ -0,0 +1,35 @@
+class Session { 
+// To be utilized when unwrapping JSON from the future servlets. Will be on the client side and be used as a buffer  
+// for client to server information needed for other components of Virtual Movie Night.
+ 
```

Comment:
These are > 80 characters, please fix

Reply:
The comment can't be more than 80 characters?

Reply:
It can but when a line becomes > 80 chars, start a new line. Like this:

// Blah blah blah when it becomes more than 80
// characters, start a new line like this and then 
// like this